helm-chart/renku-graph/templates/deployment.yaml

```diff
@@ -50,6 +50,8 @@ spec:
               {{- else }}
               value: {{ printf "http://$(SELF_IP):9001" | quote }}
               {{- end }}
+            - name: JAVA_OPTS
+              value: -Xmx100m
```

Comment:
This is the initial JVM memory allocation, correct? it can get bigger?

Reply:
No, that's the max. Considering these are stateless services I think it should be enough at least for now. We can always raise it up.

But please don't merge the PR as I'm still playing with that.

Reply:
I'd really like to reduce memory usage of the containers too.

Reply:
But it's only the heap so the actual consumption can be quite a bit more. Did you check the grafana values to see if that's a reasonable setting?

Reply:
Yes, I was looking at various things. And yes, Xmx sets the heap size but I checked yesterday and I can happily run single Graph Microservice locally with 200M memory limit. I set the limits in helm charts at 250M for `token-repository` and `webhook-service` and 300M for `triples-generator` thinking that the last one needs some more memory for `renku log` calls. However, these values are very much guessed and we simply have to stress the services and watch what happens.

Reply:
👍 this is already a big improvement over the previous state!